<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AutoDeltaAligForm</class>
 <widget class="QWidget" name="AutoDeltaAligForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>300</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>300</height>
   </size>
  </property>
  <property name="font">
   <font>
    <weight>50</weight>
    <bold>false</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>AutoDeltaHedge</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>381</width>
     <height>231</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Ticker:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_Ticket">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>10</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>SPY</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_delta">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Delta:</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox_Delta">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>40</y>
      <width>61</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>10</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>P/L:</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdNumber_pl">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>10</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="digitCount">
     <number>5</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Order:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_OrderStatus">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>70</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>40</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Com:</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdNumber_com">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>40</y>
      <width>81</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QDateEdit" name="dateEditExpire">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>10</y>
      <width>121</width>
      <height>22</height>
     </rect>
    </property>
    <property name="locale">
     <locale language="English" country="Germany"/>
    </property>
    <property name="displayFormat">
     <string>dd.MM.yyyy</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="spinBox_TargetPrice">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>40</y>
      <width>71</width>
      <height>22</height>
     </rect>
    </property>
    <property name="decimals">
     <number>2</number>
    </property>
    <property name="maximum">
     <double>9999.989999999999782</double>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>80</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Opt. delta:</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdNumber_optDelta">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>80</y>
      <width>81</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="lineWidth">
     <number>1</number>
    </property>
    <property name="smallDecimalPoint">
     <bool>false</bool>
    </property>
    <property name="digitCount">
     <number>6</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>110</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Basis delta:</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdNumber_basisDelta">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>110</y>
      <width>81</width>
      <height>23</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>140</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Delta sum:</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdNumber_SumDelta">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>140</y>
      <width>81</width>
      <height>23</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_OptionType">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>40</y>
      <width>41</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>Call</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_isBuy">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>70</y>
      <width>72</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>Buy</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="buttonStartStop">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>260</y>
     <width>80</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Start</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Sell">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>260</y>
     <width>80</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Sell</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Buy">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>260</y>
     <width>80</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Buy</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
